src/index.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Route, BrowserRouter } from 'react-router-dom';
+import Quotes from './pages/Quote';
+import Home from './pages/Home';
+import Navbar from './components/Navbar';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Navbar />
+      <Route exact path="/">
+        <Home />
+      </Route>
+      <Route path="/calculator">
+        <Calculator />
+      </Route>
+      <Route path="/quote">
+        <Quotes />
+      </Route>
+    </BrowserRouter>
```

Comment:
- [ ] I would recommend using `switch` inside `BrowserRouter`, it's because in this case when we open `calculator` it will render other components like `quote` as well in this case.

Why the `switch` is important 
```
The `<Switch />` component will only render the first route that matches/includes the path.
Once it finds the first route that matches the path, it will not look for any other matches.
Not only that, it allows for nested routes to work properly, which is something that <Router />
will not be able to handle.

```